packages/web-components/index.js

```diff
@@ -2,6 +2,10 @@
 
 export { makeAgoricKeplrConnection } from './src/keplr-connection/KeplrConnection.js';
 export { makeAgoricWalletConnection } from './src/wallet-connection/walletConnection.js';
+export {
+  agoricRegistryTypes,
+  agoricConverters,
+} from './src/wallet-connection/signerOptions.js';
```

Comment:
Nice, glad to see to see these exported. Should we also update the README to include this?

Reply:
We should but I think I'd like to land the react-components stuff first so we can have a more cohesive view/example